server/src/infra/repositories/communication.repository.ts

```diff
@@ -34,7 +41,7 @@ export class CommunicationRepository implements OnGatewayConnection, OnGatewayDi
   }
 
   send(event: CommunicationEvent, userId: string, data: any) {
-    this.server.to(userId).emit(event, JSON.stringify(data));
+    this.server.to(userId).emit(event, data);
```

Comment:
@alextran1502 @shenlong-tanwen - I noticed that `broadcast` did not use `JSON.stringify`, while `send` did. I changed both to omit calling `JSON.stringify`, can one of you help me validate the mobile app events still work properly with this change? Thanks.